tests/dualtor/test_orchagent_mac_move.py

```diff
@@ -88,8 +88,13 @@ def test_mac_move(
     tbinfo, tunnel_traffic_monitor, vmhost          # noqa F811
 ):
     tor = rand_selected_dut
-    ptf_t1_intf = random.choice(get_t1_ptf_ports(tor, tbinfo))
-    ptf_t1_intf_index = int(ptf_t1_intf.strip("eth"))
+    portchannels = None
```

Comment:
what's the purpose of this code section? I didn't see any description in the commit message